csrc/tma.cpp

```diff
@@ -356,25 +355,43 @@ std::vector<PolymorphicValue> kir::EncodeTensorMapTiled::evaluate(
     auto bounding_box_inner_dim = box_dim.at(0) * elem_size;
     switch (swizzle) {
       case CU_TENSOR_MAP_SWIZZLE_32B:
+        // bounding_box_inner_dim == 32
         NVF_ERROR(
```

Comment:
It seems there's redundancy in the error checks related to `bounding_box_inner_dim`. The two checks appear to be verifying that this value is exactly 32, but this could be more directly and efficiently validated with a single check.

Reply:
Yes, agree. I intentionally uses two checks because they are due to different reasons.

Reply:
Got you! I usually alter the err msg based on different reasons.